alex_w/server.js

```diff
@@ -0,0 +1,109 @@
+var express = require("express");
+var path = require("path");
+var app = express();
+var mongoose = require('mongoose');
+var bodyParser = require('body-parser');
+
+mongoose.connect('mongodb://localhost/mongoose_dashboard');
+
+var PandaSchema = new mongoose.Schema({
+ name: String,
+ age: Number
+})
+mongoose.model('Panda', PandaSchema); 
+var Panda = mongoose.model('Panda') 
```

Comment:
We can combine lines 13 and 14 into 'var Panda = mongoose.model('Panda', PandaSchema);'